Tests/Imagine/Filter/Loader/UpscaleFilterLoaderTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Tests\Filter;
+
+use Imagine\Image\Box;
+use Liip\ImagineBundle\Imagine\Filter\Loader\UpscaleFilterLoader;
+use Liip\ImagineBundle\Tests\AbstractTest;
+
+/**
+ * Test cases for UpScaleFilterLoader class.
+ *
+ * @covers Liip\ImagineBundle\Imagine\Filter\Loader\UpScaleFilterLoader
```

Comment:
Casing on the class-name is a bit dodgy!
Also can the new test incorporated into ScaleLoader so that all the tests for this class are in one place?